src/catalog/products.md

```diff
@@ -6,7 +6,7 @@ redirect_from:
 
 All products in the catalog are accessible from the [Products]({% link catalog/products.md %}) page, where you can create new products and edit existing ones. For a multi-site installation, each website can offer a different selection of products for sale from the same catalog.
 
-The Products grid lists all products in the catalog, indicates the website(s) where they are available, and if they are currently enabled for sale. In Magento Commerce for B2B installations with [Shared Catalog]({% link catalog/catalog-shared.md %}) enabled, the grid includes a column that indicates which products have alternate discount pricing in a shared catalog.
+The Products grid lists all products in the catalog, indicates the website(s) where they are available, and if they are currently enabled for sale. In Magento Commerce for B2B installations with the [Shared Catalog]({% link catalog/catalog-shared.md %}) enabled, the grid includes a column that indicates which products have alternate discount pricing in a shared catalog.
```

Comment:
Remove the "the" you added. Change Magento Commerce for B2B to a variable.

Reply:
You mean adding it like this: `<span class="b2b-only">Magento Commerce for B2B</span>` ?

Reply:
I'm not sure how variables work in merchdocs, but in devdocs, it would be `{{ site.data.var.b2b }}`.

Reply:
We don't use the `site.data` variable for internal UG links.